src/ui/public/index_patterns/_index_pattern.js

```diff
@@ -476,7 +479,7 @@ export function IndexPatternProvider(Private, config, Promise, confirmModalPromi
                 }
 
                 if (unresolvedCollision) {
-                  toastNotifications.addDanger('Unable to write index pattern! Refresh the page to get the most up to date changes for this index pattern.'); // eslint-disable-line max-len
+                  toastNotifications.addDanger(i18n('common.ui.indexPattern.create.unableWrite.label', { defaultMessage: 'Unable to write index pattern! Refresh the page to get the most up to date changes for this index pattern.' })); // eslint-disable-line max-len
```

Comment:
Can we extract the text into its own variable to avoid this really long line?

Reply:
There is a command `node scripts/extract_default_translations`  which creates `en.json` with default translations by static code analysis. The documentation for this tool is in progress, @LeanidShutau is working on it.